docs/CLI.md

```diff
@@ -0,0 +1,142 @@
+# covalent cli
+
+## Overview
+
+CLI (Command Line Interface) for Covalent.
+
+
+### Usage
+
+To run the covalent tools cli do the following:
+
+* Clone this repository
+* npm i
+* npm run build
+* npm install -g
+
+#### The following commands are available in the CLI
+
+#### Help Usage
+```
+cov -h
+  Usage: cov [options]
+
+
+  Options:
+
+    -V, --version           output the version number
+    -v, --all-versions      output the version information
+    -g, --gen-e2e-report    generate the e2e coverage report
+    -s, --start-e2e-proxy   start the e2e proxy
+    -e, --e2e-for-coverage  runs the end to end tests against the e2e proxy
```

Comment:
similary, with `-g` `-s` and `-e` just not sure the usefulness of having the short form flags.